pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -201,10 +201,10 @@ type CephBlockPool struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata"`
 	Spec              PoolSpec `json:"spec"`
+	Status            *Status  `json:"status"`
```

Comment:
Does it still make sense to add status if we're adding conditions (#4336)?

Reply:
yes, we can still keep it. The motivation for adding it was, in openshift's console it shows status as unknown for these CRs